### ISSUE TITLE
ERB support within comments?

### ISSUE THREAD

**Issue by gjtorikian** [allmarkedup/lookbook#104]
Thanks for this great project.

I was wondering if it's possible to use ERB within comments themselves, to facilitate dynamic parameter lists. For example, the following is valid in plain YARD:

```ruby
require "yard/renders_one_handler"


class BaseButtonComponentPreview < ViewComponent::Preview
  TYPE_OPTIONS = [:button, :reset, :submit]
  # @param tag [Symbol] <%= one_of(BaseButton::TYPE_OPTIONS) %>
   def default(tag: 
    # ...
    end
end
```

However, the following does not work in Lookbook:

```
# @param tag [Symbol] select <%= one_of(BaseButton::TYPE_OPTIONS) %>
```

I also tried formatting the comment in the expected format, but that didn't work either:

```ruby
 <%= BaseButton::TYPE_OPTIONS).map { |i| [i.to_s, i] } %>
 ```
 
 (By "it doesn't work," I mean the dropdown is simply never filled.) It looks to me like ERB just isn't supported?

**Comment by allmarkedup** [comment]
Hey @gjtorikian, it's taken a little while to get round to it but I've just released v1.1.0 that includes support for evaluating Ruby statements to generate `@param` options dynamically. It's not using ERB but achieves the same end result.

Docs are here: https://lookbook.build/guide/previews/params/#param-options - I'm going to close this issue down now but let me know how you get on if you get a chance to try it out!

**Comment by gjtorikian** [comment]
Awesome, thank you!
